apiclient/helpers.go

```diff
@@ -162,6 +162,42 @@ func (c *HTTPclient) WaitUntilTxIsMined(ctx context.Context,
 	}
 }
 
+// WaitUntilElectionKeys waits until the election has published its encryption keys,
+// and returns them along with keyIndexes.
+func (c *HTTPclient) WaitUntilElectionKeys(pid []byte,
+) (keys []types.HexBytes, keyIndexes []uint32, err error) {
+	waitTimeout := 10 * time.Second
```

Comment:
Timeout is too narrow.

Reply:
good catch, actually this whole func needed refactoring.